### PR TITLE
[master<] Changelog for Lab 2.7.1

### PR DESCRIPTION
**Pull request by vpavicic**
### Description

Created a changelog for Lab 2.7.1

### CODE REVIEW

**Review by cizl — CHANGES_REQUESTED** [review]
We added 2 more fixes:

* Query execution choice now closes on click as expected.
* Fixed bug and improved behavior when running a query that has style errors.

And
* Renamed System Default Style to System Style.

**Review by cizl — APPROVED** [review]
Nice! :fire:

File: lab/changelog.md [hunk 1]
```diff
@@ -4,6 +4,26 @@ title: Changelog
 sidebar_label: Changelog
 ---
 
+## Lab v2.7.1 - Jul 05, 2023
+
+### Improvements
+
+- System Default style has been renamed to System Style. 
+- If you run a query that has errors in the Graph Style Script code you can
+  decide to run it with the System Style.
```
Comment (vpavicic) [inline lab/changelog.md:13]:
```suggestion
  decide to run it using the System Style. 
```

File: lab/changelog.md [hunk 2]
```diff
@@ -4,6 +4,26 @@ title: Changelog
 sidebar_label: Changelog
 ---
 
+## Lab v2.7.1 - Jul 05, 2023
+
+### Improvements
+
+- System Default style has been renamed to System Style. 
+- If you run a query that has errors in the Graph Style Script code you can
```
Comment (vpavicic) [inline lab/changelog.md:12]:
```suggestion
- If you run a query that has errors in the Graph Style Script code, you can
```

**Pull request merged**